sdk/sdk-view/src/test/java/mil/emp3/api/CameraLookAtTest.java

```diff
@@ -23,18 +25,33 @@
 
     @Before
     public void setUp() throws Exception{
-        init();
+        setupSingleMap(TAG);
         camera = new Camera();
         lookAt = new LookAt();
     }
 
     @After
     public void cleanUp() throws Exception{
-
+        mapInstance.cleanQueues();
     }
 
     @Test
     public void cameraTests() {
```

Comment:
this test has a lot of asserts. It is typically good convention to have only one assert per test (keeps test short, readable, and testing only one functionality), but if this is not possible could you please add comments within the test to add more clarity to what it is testing, since it is long

Reply:
It does not have a 'lot' it only has two asserts, one to initialize the camera settings and one to modify and apply.

Reply:
Okay well I would be okay if you just added some documentation on this test :)